packages/ucp/docs/getting-started.md

```diff
@@ -0,0 +1,361 @@
+// instantiate koreografeye policy executor
+const policyExecutor = new PolicyExecutor(plugins)
+// ucon storage
+const uconRulesStorage = new MemoryUCRulesStorage();
+// load N3 Rules
+const response = await fetch('https://raw.githubusercontent.com/woutslabbinck/ucp-enforcement/main/rules/data-crud-rules.n3'); // loading from the github repo
+const n3Rules: string[] = [await response.text()]
+// instantiate the enforcer using the policy executor,
+const ucpEvaluator = new UcpPatternEnforcement(uconRulesStorage, n3Rules, new EyeJsReasoner([
+        "--quiet",
+        "--nope",
+        "--pass"]), policyExecutor)
+```
+
+At this point, the engine is ready to be used. Which means that now you can use the `calculateAccessModes` function to request the **grants** for following *action request*: "Ruben wants to know the age of Wout".
+
+```ts
+const accessModes = await ucpEvaluator.calculateAccessModes({
+    subject: "https://pod.rubendedecker.be/profile/card#me",
+    action: ["http://www.w3.org/ns/auth/acl#Read"],
+    resource: "urn:wout:age",
+    owner: "https://pod.woutslabbinck.com/profile/card#me"
+});
+console.log(accessModes);
+```
+
+Unfortunately, this results into an empty list `[]`. Which means no **grants** are given and thus Ruben cannot know the age of Wout.
+
+The reason for this is very simple, there is **no** *Usage Control Policy* in the storage.
+
+This can however be resolved by simply adding such a policy to the **Usage Control Rule Storage**:
+
+```ts
+@prefix odrl: <http://www.w3.org/ns/odrl/2/> .
+@prefix : <http://example.org/usageControlRule> .
+
+:permission
+  a odrl:Permission ;
+  odrl:action odrl:read ;
+  odrl:target <urn:wout:age> ;
+  odrl:assignee <https://pod.rubendedecker.be/profile/card#me> ;
+  odrl:assigner <https://pod.woutslabbinck.com/profile/card#me> .
+```
+
+To add this rule to the storage, the following code can be used:
+
+```ts
+const ucr = `@prefix odrl: <http://www.w3.org/ns/odrl/2/> .
+@prefix : <http://example.org/usageControlRule> .
+
+:permission
+  a odrl:Permission ;
+  odrl:action odrl:read ;
+  odrl:target <urn:wout:age> ;
+  odrl:assignee <https://pod.rubendedecker.be/profile/card#me> ;
+  odrl:assigner <https://pod.woutslabbinck.com/profile/card#me> .
+    `
+const policyStore = await turtleStringToStore(ucr);
+await uconRulesStorage.addRule(policyStore);
+```
+
+From now on, when the access modes are calculated again, the following grants are received:
+
+```sh
+[ 'http://www.w3.org/ns/auth/acl#Read' ]
+```
+
+The full code sample for this example can be found in [appendix I](#appendix-I-Full-code-snippet)
+
+### Temporal Policy engine
+
+TODO:
+
+### Engine with explanation
+
+First an elaboration is given for what is meant by explanation.
+Note: after implementation, similarities have been found with the idea of **ODRL Evaluator** from the [ODRL Formal Semantics Community Group](https://w3c.github.io/odrl/formal-semantics/).
```

Comment:
Not sure if we should mention that here in the docs, if we're not going to elaborate on it.

Reply:
I'll comment it out, it's basically a reminder to myself.